dropwizard/src/main/java/com/linecorp/armeria/server/dropwizard/ArmeriaServerFactory.java

```diff
@@ -0,0 +1,222 @@
+    }
+
+    public boolean isJerseyEnabled() {
+        return jerseyEnabled;
+    }
+
+    public void setJerseyEnabled(final boolean jerseyEnabled) {
+        this.jerseyEnabled = jerseyEnabled;
+    }
+
+    public AccessLogWriterFactory getAccessLogWriter() {
+        return accessLogWriter;
+    }
+
+    /**
+    * Sets an {@link AccessLogWriter} onto this ServerFactory.
+    *
+    * @param accessLogWriter - an instance of a {#link AccessLogWriter}
+    */
+    public void setAccessLogWriter(@Valid final AccessLogWriterFactory accessLogWriter) {
+        this.accessLogWriter = Objects.requireNonNull(accessLogWriter,
+                                          "server[type=\"armeria\"].accessLogWriter");
+    }
+
+    public Size getMaxRequestLength() {
+        return maxRequestLength;
+    }
+
+    public void setMaxRequestLength(final Size maxRequestLength) {
+        this.maxRequestLength = maxRequestLength;
+    }
+
+    @JsonIgnore
+    public ServerBuilder getServerBuilder() {
+        return serverBuilder;
+    }
+
+    @Override
+    public Server build(final Environment environment) {
+        this.printBanner(environment.getName());
+        final MetricRegistry metrics = environment.metrics();
+        final ThreadPool threadPool = this.createThreadPool(metrics);
+        final Server server = this.buildServer(environment.lifecycle(), threadPool);
+
+        final JerseyEnvironment jersey = environment.jersey();
+        if (!isJerseyEnabled()) {
+            jersey.disable();
+        }
+
+        final Handler applicationHandler = this.createAppServlet(server,
+                jersey,
+                environment.getObjectMapper(),
+                environment.getValidator(),
+                environment.getApplicationContext(),
+                environment.getJerseyServletContainer(),
+                metrics);
+        final Handler adminHandler = this.createAdminServlet(server,
+                environment.getAdminContext(),
+                metrics,
+                environment.healthChecks());
+        final ContextRoutingHandler routingHandler = new ContextRoutingHandler(ImmutableMap.of(
+                getApplicationContextPath(), applicationHandler,
+                getAdminContextPath(), adminHandler));
+        final Handler gzipHandler = this.buildGzipHandler(routingHandler);
+        server.setHandler(this.addStatsHandler(this.addRequestLog(server, gzipHandler, environment.getName())));
+
+        setupArmeria(server, metrics);
+
+        return server;
+    }
+
+    private void setupArmeria(final Server server, final MetricRegistry metricRegistry) {
+        LOGGER.debug("Building Armeria Server");
+        final JettyService jettyService = JettyService.forServer(server);
+        this.serverBuilder = com.linecorp.armeria.server.Server.builder()
+                .meterRegistry(DropwizardMeterRegistries.newRegistry(metricRegistry))
+                .serviceUnder("/", jettyService);
```

Comment:
This alone appears to work for every Jersey resource as well as the AdminServlet added above here in `build()`... 

Not sure what happens when Jersey and Armeria have the same paths exposed

Reply:
The server added first will always have precedence. In an obvious case, it refuses to start and ask a user to fix the problem.

Reply:
The rule of thumb is to call the methods like `serviceUnder` as late as possible so that more specific routes are picked up first. In this case, you could add `jettyService` after all other Armeria services are bound.